test/fixtures/inline-loader-syntax.js

```diff
@@ -0,0 +1,5 @@
+import css from 'Test.module.scss!=!./inline-loader-syntax.css';
```

Comment:
I would expect that css-loader treat this resource as a `module` but it does not have access to the matchResource so can't even use it if it wanted to